client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -99,7 +85,19 @@ export const HoldList = ( {
 				{ count: holds.length }
 			) } />
 			<Card className="eligibility-warnings__hold-list">
-				{ map( holds, hold =>
+				{ isPlaceholder &&
+					<div>
+						<div className="eligibility-warnings__hold">
+							<Gridicon icon="notice-outline" size={ 24 } />
+							<div className="eligibility-warnings__message"></div>
+						</div>
+						<div className="eligibility-warnings__hold">
+							<Gridicon icon="notice-outline" size={ 24 } />
+							<div className="eligibility-warnings__message"></div>
+						</div>
+					</div>
+				}
```

Comment:
not bad! we might want to investigate our placeholder or skeleton styling because we have certain classes that will make this stuff fade in and out